File: java/src/test/java/kms/notice/TestGetNoticeList.java
package kms.notice;

import dev.spiralmoon.maplestory.api.kms.MapleStoryApi;
import dev.spiralmoon.maplestory.api.kms.dto.notice.NoticeListDTO;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

@DisplayName("getNoticeList")
public class TestGetNoticeList {
    private static final String apiKey = System.getProperty("API_KEY_KMS"); // Your API Key
    private static final MapleStoryApi api = new MapleStoryApi(apiKey);

    @Test
    @DisplayName("success: getNoticeList")
    void getNoticeList() {
        NoticeListDTO response = api.getNoticeList().join();
        System.out.println(response.toString());
    }
}

File: csharp/MapleStory.Test/KMS/Notice/TestGetNoticeList.cs
using NUnit.Framework;
using MapleStory.OpenAPI.KMS;

namespace MapleStory.Test.KMS.Notice
{
    [TestFixture]
    public class TestGetNoticeList
    {
        private static readonly string apiKey = EnvConfig.Get("API_KEY_KMS"); // Your API Key
        private static readonly MapleStoryAPI api = new MapleStoryAPI(apiKey);

        [Test, Description("success: GetNoticeList")]
        public async Task GetNoticeList()
        {
            var response = await api.GetNoticeList();
            Console.WriteLine(response.ToJson());
        }
    }
}
